tools/scripts/generate_binary_build_matrix.py

```diff
@@ -17,7 +17,7 @@
 import os
 import sys
 
-from typing import Dict, List, Optional, Tuple, Any, Callable
+from typing import Any, Callable, Dict, List, Optional, Tuple
```

Comment:
Why this change is needed?

Reply:
Disabled auto-formatted in vscode. Sorry about this